sentences/pt-br/_common.yaml

```diff
@@ -364,6 +364,8 @@ expansion_rules:
   falta: "(falta|resta)"
   alarme: "(alarme[s]|cronômetro[s]|timer[s]|temporizador[es])"
   pausar: "(pausa[r]|pause)"
+  cortina: "(cortina[s]|curtinha[s]|persiana[s])"
```

Comment:
Never heard of "Curtinha", is it some regional term?

Reply:
Nope, it's a common mistake from STT providers (including the one home assistant cloud uses) when you say "cortina", so this will cover those cases as well